README.md

```diff
@@ -15,6 +15,13 @@ The MarkLogic Connector for Hadoop is an extension to Hadoop’s MapReduce frame
 
 ## Release Notes
 
+### What's New in mlcp 11.2.0
+
+- Upgraded Avro, Commons-cli, Commons-compress, Commons-io, Xml-apis, Jakarta.xml.soap-api, Jcl-over-slf4, and Commons-logging-api libraries to mitigate security vulnerabilities.
```

Comment:
The first two bullet points should be exclude all transitive dependencies, right?

Reply:
Resolved. Thanks